app/src/main/java/com/marvelapp/MainActivity.kt

```diff
@@ -0,0 +1,23 @@
+package com.marvelapp
+
+import android.os.Bundle
+import android.view.animation.AnimationUtils
+import androidx.appcompat.app.AppCompatActivity
+import com.marvelapp.databinding.ActivityMainBinding
+
+class MainActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityMainBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        binding.marvelLogoSplashImageView.startAnimation(
```

Comment:
If you are using mvvm, maybe in this line you need to call the viewmodel to set the state. @maumuromurito